app/src/main/java/com/github/se/eventradar/ui/home/Home.kt

```diff
@@ -132,12 +135,12 @@ fun HomeScreen(
 
     SearchBarAndFilter(
         searchQuery = uiState.searchQuery,
-        onSearchQueryChange = { uiState.searchQuery = it },
```

Comment:
Search and filter was my job, so I have made these changes already in my incoming PR. But since you've created tests for them already, let's keep them here. 

I will update your functions/tests related to search and filter with my work where necessary, since I have already tested on my end to ensure it works.